src/popup/utils/transactionTokenInfoResolvers.ts

```diff
@@ -1,24 +1,65 @@
 import BigNumber from 'bignumber.js';
+import {
+  IToken,
+  ITokenList,
+  ITransaction,
+  TxFunctionParsed,
+} from '../../types';
 import { AETERNITY_SYMBOL } from './constants';
 
-const defaultToken = {
+/* eslint-disable no-unused-vars */
+export interface ITokenResolved extends Partial<IToken> {
+  amount?: number
+  isAe?: boolean
+  isPool?: boolean
+  isReceived?: boolean
+}
+
+interface TransactionResolverReturnData {
+  tokens: ITokenResolved[]
+  sender?: any
+  recipient?: any
+}
+
+type TransactionResolver = (
+  transaction: ITransaction,
+  tokens: ITokenList | null
+) => TransactionResolverReturnData;
+
+type TransactionResolverGenerator = (
+  tokenAMapper: (t: ITransaction) => any,
+  tokenBMapper: (t: ITransaction) => any,
+  poolTokenAmountMapper?: (t: ITransaction) => any,
+  liquidityMethod?: 'add' | 'remove'
+) => TransactionResolver;
+/* eslint-enable no-unused-vars */
```

Comment:
is it needed here?

Reply:
You mean the eslint enable? It enables again what was disabled in line 11.

Reply:
Sorry, I misread that. No issues then